Formula/serf.rb

```diff
@@ -108,13 +39,23 @@ def install
     arch = MacOS.prefer_64_bit? ? "amd64" : "386"
     ENV["XC_ARCH"] = arch
     ENV["XC_OS"] = "darwin"
-    ENV.prepend_create_path "PATH", gopath/"bin"
 
     Language::Go.stage_deps resources, gopath/"src"
 
+    ENV.prepend_create_path "PATH", gopath/"bin"
+    cd gopath/"src/github.com/mitchellh/gox" do
+      system "go", "build"
+      (gopath/"bin").install "gox"
+    end
+
     cd gopath/"src/github.com/hashicorp/serf" do
+      # We've already handled downloading/building these
+      # and don't need to repeat the step.
+      inreplace "GNUmakefile", "bin:: tools", "bin::"
```

Comment:
I'll double check if this is needed actually. I put this in first, and then messed with the `gox` stuff a bit more, it might be redundant now. Upstream do have some sane logic around finding `gox` on system first.